import "goby/common/protobuf/option_extensions.proto";
import "goby/common/protobuf/app3.proto";
import "goby/middleware/protobuf/interprocess_config.proto";
import "dccl/protobuf/option_extensions.proto";

package goby.protobuf;

message TerminateConfig
{
    option (dccl.msg).unit_system = "si";
    optional App3Config app = 1;
    optional InterProcessPortalConfig interprocess = 2;
    repeated string target_name = 3;
    repeated uint32 target_pid = 4;

    optional float response_timeout = 5
        [default = 10, (dccl.field).units.base_dimensions = "T"];
}
